docker/docker-compose.1804.50.yaml

```diff
@@ -16,9 +16,9 @@ services:
   integration-tests:
     image: swift-nio-http2:18.04-5.0
     environment:
-      - MAX_ALLOCS_ALLOWED_create_client_stream_channel=70010
+      - MAX_ALLOCS_ALLOWED_create_client_stream_channel=72010
       - MAX_ALLOCS_ALLOWED_hpack_decoding=5050
-      - MAX_ALLOCS_ALLOWED_client_server_request_response=360000
+      - MAX_ALLOCS_ALLOWED_client_server_request_response=374000
```

Comment:
This increase (14 per test run, approximately) is for channel atomics and the new futures and their callbacks.